templates/webapi/test/overview.html.ep

```diff
@@ -42,8 +42,12 @@
         <div class="card-body">
             % my $allow_commenting = @$job_ids && current_user;
             % if ($allow_commenting) {
-            <button type="button" class="btn btn-secondary btn-circle btn-sm trigger-edit-button" onclick="showAddCommentsDialog()" title="Add comments">
-                <i class="fa fa-comment" aria-hidden="true"></i>
+            <button type="button" class="btn btn-secondary btn-circle btn-sm
+            trigger-edit-button" onclick="showAddCommentsDialog()" title="Restart or comment jobs">
+                <span class="fa-stack" style="font-size: 0.9em" >
+		  <i class="fa fa-comment fa-stack-2x text-danger-info" aria-hidden="true"></i>
+		  <i class="fa fa-undo fa-stack-1x fa-inverse text-danger" aria-hidden="true"></i>
```

Comment:
No need to show the undo icon in red. Just the gray from the background should be ok if that's enough contrast

Reply:
![image](https://github.com/user-attachments/assets/5b2f32c8-5986-4066-a23f-8793a34a5f41)